hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/ha/GrpcOMFailoverProxyProvider.java

```diff
@@ -59,10 +63,14 @@ public class GrpcOMFailoverProxyProvider<T> extends
   public static final Logger LOG =
       LoggerFactory.getLogger(GrpcOMFailoverProxyProvider.class);
 
+  private final long protocolVer;
+
   public GrpcOMFailoverProxyProvider(ConfigurationSource configuration,
+                                     UserGroupInformation ugi,
                                      String omServiceId,
                                      Class<T> protocol) throws IOException {
-    super(configuration, omServiceId, protocol);
+    super(configuration, ugi, omServiceId, protocol);
+    this.protocolVer = RPC.getProtocolVersion(protocol);
```

Comment:
`protocolVer` is unused.

Reply:
Addressed in [c52deae](https://github.com/apache/ozone/pull/7268/commits/c52deaec7d51ec9adea755d75d83e82bcdd74e3d)